jquery.minimalect.js

```diff
@@ -165,6 +166,19 @@ module.exports = function(jQuery) {
 					});
 				}
 
+				// Event handler for filtering functionality on keypress
+				var filterHandler = function(e){
```

Comment:
Yeah this document uses 4-spaces and indents. Don't know if we should switch it to 2. And if we do should be another PR so these changes don't get lost in that :smile: